README.md

```diff
@@ -209,12 +209,15 @@ and to some degree decouples the test setup from the test execution.
 ### Creating a new Retry Test
 
 The following cURL request will create a Retry Test resource which emits a 503
-when a buckets list operation is received by the testbench with the returned
+when a buckets list operation is received by the testbench JSON server with the returned
 retry test ID.
 
+Note: A Retry Test resource can only apply to one transport, either JSON or GRPC. It defaults
+to "JSON" if not specified.
+
 ```bash
 curl -X POST "http://localhost:9000/retry_test" -H 'Content-Type: application/json' \
-     -d '{"instructions":{"storage.buckets.list": ["return-503"]}}'
+     -d '{"instructions":{"storage.buckets.list": ["return-503"]}, "transport": "JSON"}'
```

Comment:
IMO the options should really be HTTP or gRPC. The server does support (a few) XML endpoints for the retry test API currently and I assume those would still be supported.

Reply:
The few XML endpoints are still supported. Happy to update this to HTTP